package.json

```diff
@@ -7,7 +7,7 @@
   "types": "./dist/typings/contentful-management.d.ts",
   "module": "./dist/es-modules/contentful-management.js",
   "engines": {
-    "node": ">=6"
+    "node": ">=16"
```

Comment:
Node 14 isn't maintenance EOL until [2023-04-30](https://nodejs.org/en/about/releases/), and moreover is the latest supported node version on AWS Lambda - [this comment on the AWS repo suggests that v16 might be coming May 2022](https://github.com/aws/aws-lambda-base-images/issues/14#issuecomment-1100211561), but until then, any lambda-based apps will be unable to use this.

(Certainly think we need to move past node 6, though, and probably past 12 as well , which is maintenance EOL the end of this month.)

Reply:
Valid point. Using v14 then. Thanks

Reply:
Should we run the tests with node 14 as well?

Reply:
It was already at v16 before my PR so I can downgrade it but I don't know if version 16 was used intentionally or not.

Reply:
Using v14. As discussed, circleci orb will use v14 thanks [to its node/install command ](https://circleci.com/developer/orbs/orb/circleci/node#usage-install_nodejs)

Reply:
![Screenshot 2022-04-20 at 13 58 04](https://user-images.githubusercontent.com/4171202/164225480-91639500-b2da-4b41-ae41-d355d2ab3cca.png)

Reply:
👍 

We should now be able to use `cimg/base:stable` and downgrade the lockfile

Reply:
Fine. I kept the locked version just to avoid sudden bumps behind the "stable" tag. But make sense to use stable as well